federated-xgboost/README.md

```diff
@@ -3,10 +3,29 @@ Federated learning allows multiple parties to collaboratively learn a shared mod
 
 In the federated setting, a central party has a basic model that is initially broadcast to all parties. Each party locally trains the model with its own data, then sends a summary of the updates to the model back to the central party. In the decision tree case, parties would be sending the best local feature splits back to the central party. The central party then aggregates all updates, updates its own model with the aggregated update, and broadcasts the newly updated model to all parties. This process is then repeated over and over.
 
-![federated diagram](./images/federated-xgboost-diagram.png)
-
 This project extends the existing XGBoost gradient boosting machine learning framework to enable training models in the federated setting. This work is being actively contributed to and is still under development.
 
+[QUICKSTART](#quickstart)
+
+![federated diagram](./images/federated-xgboost-diagram.png)
+
+### Finding a split
+
+![diagram 1](./images/detail_1.png)  
+
+![diagram 2](./images/detail_2.png)  
+
+![diagram 3](./images/detail_3.png)  
+
+![diagram 4](./images/detail_4.png)  
+
```

Comment:
Can you enumerate the steps to show that one comes after another? Also is there any way to do two diagrams in a row? So step 1 and step 2 would be in one row, and step 3 and 4 underneath that? I think it'd look better in the README. Thanks!